include/macros.h

```diff
@@ -129,4 +129,12 @@ extern GraphicsContext* __gfxCtx;
 
 #define VTX_T(x,y,z,s,t,cr,cg,cb,a) { { x, y, z }, 0, { s, t }, { cr, cg, cb, a } }
 
+#ifdef NDEBUG
+#define ASSERT(cond, msg, file, line) ((void) 0)
+#elif defined(REAL_ASSERT_MACRO)
+#define ASSERT(cond, msg, file, line) ((cond) ? (void) 0 : __assert(#cond, __FILE__, __LINE__))
+#else
+#define ASSERT(cond, msg, file, line) ((cond) ? (void) 0 : __assert(msg, file, line))
```

Comment:
Very minor but I think this would be a bit cleaner
```suggestion
#define ASSERT(cond, msg, file, line) ((void)0)
#elif defined(REAL_ASSERT_MACRO)
#define ASSERT(cond, msg, file, line) ((cond) ? ((void)0) : __assert(#cond, __FILE__, __LINE__))
#else
#define ASSERT(cond, msg, file, line) ((cond) ? ((void)0) : __assert(msg, file, line))
```